site/content/how-to/traffic-management/redirects-and-rewrites.md

```diff
@@ -222,6 +244,285 @@ Server name: coffee-6db967495b-twn6x
 URI: /prices?test=v1&test=v2
 ```
 
+---
+
+## RequestRedirect example
+
+This example demonstrates how to redirect the traffic to a new location for a tea and soda applications. We'll examine how request redirects works using the `RequestRedirect` filter by creating four HTTPRoutes: two for redirecting requests to a new location and other two to define the destination location blocks for the redirected traffic.
+
+---
+
+### Setup
+
+Create the **tea** and **soda** application in Kubernetes by copying and pasting the following block into your terminal:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tea
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tea
+  template:
+    metadata:
+      labels:
+        app: tea
+    spec:
+      containers:
+      - name: tea
+        image: nginxdemos/nginx-hello:plain-text
+        ports:
+        - containerPort: 8080
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: tea
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+    protocol: TCP
+    name: http
+  selector:
+    app: tea
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: soda
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: soda
+  template:
+    metadata:
+      labels:
+        app: soda
+    spec:
+      containers:
+      - name: soda
+        image: nginxdemos/nginx-hello:plain-text
+        ports:
+        - containerPort: 8080
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: soda
+spec:
+  ports:
+  - port: 80
+    targetPort: 8080
+    protocol: TCP
+    name: http
+  selector:
+    app: soda
+EOF
+```
+
+This will create the **tea** and **soda** service and a deployment. Run the following command to verify the resources were created:
```

Comment:
```suggestion
This will create the **tea** and **soda** services and deployments. Run the following command to verify the resources were created:
```